packages/app/components/stackbox/FrequencyOptionsCard.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import { ChangeEvent, useEffect, useState } from "react";
+
+import { twMerge } from "tailwind-merge";
+
+import { FREQUENCY_OPTIONS } from "@/models";
+import { BodyText, RadioButton, TextInput } from "@/ui";
+
+interface FrequencyOptionsCardProps {
+  frequency: FREQUENCY_OPTIONS;
+  setEndDate: (date: Date) => void;
+}
+
+const defaultFrequencyOptions = {
+  [FREQUENCY_OPTIONS.hour]: ["12", "24", "72"],
+  [FREQUENCY_OPTIONS.day]: ["7", "15", "30"],
+  [FREQUENCY_OPTIONS.week]: ["5", "10", "20"],
+  [FREQUENCY_OPTIONS.month]: ["3", "6", "12"],
+};
+const maxCustomFrequencies = {
+  [FREQUENCY_OPTIONS.hour]: 96,
+  [FREQUENCY_OPTIONS.day]: 365,
+  [FREQUENCY_OPTIONS.week]: 52,
+  [FREQUENCY_OPTIONS.month]: 36,
```

Comment:
I would say that we don't want such a big stacks (as they can be low values), so I would reduce at least the months to 24.

```suggestion
  [FREQUENCY_OPTIONS.month]: 24,
```

Reply:
I like it, so basically we top up to 1 year DCA orders in almost every frequency as max (unless the user wants to set them via Date pickers)